Sources/CodeScanner/CodeScanner.swift

```diff
@@ -163,11 +164,14 @@ public struct CodeScannerView: UIViewControllerRepresentable {
             view.backgroundColor = UIColor.black
             captureSession = AVCaptureSession()
 
-            guard let videoCaptureDevice = AVCaptureDevice.default(for: .video) else { return }
+            if videoCaptureDevice == nil{
+                return
+            }
+
             let videoInput: AVCaptureDeviceInput
 
             do {
-                videoInput = try AVCaptureDeviceInput(device: videoCaptureDevice)
+                videoInput = try AVCaptureDeviceInput(device: videoCaptureDevice!)
             } catch {
                 return
             }
```

Comment:
With the above change to a guard statement, you don't need to force-unwrap `!` any more. Additionally, since you're only interested in the successful part of the `try/catch` block, you can use `try?` to make this more readable and shorter:

```suggestion
            guard let videoInput = try? AVCaptureDeviceInput(device: videoCaptureDevice) else { return }
```